ged2dot.py

```diff
@@ -949,18 +958,22 @@ def get(self, what: str, fallback: str = configparser._UNSET) -> str:  # type: i
 
 
 def main() -> None:
+    if not os.path.exists("ged2dotrc"):
+        sys.stderr.write("Fatal: ged2dotrc configuration file doesn't exist.\nCreate a config file similar to test/screenshotrc, name it ged2dotrc and continue.\n")
+        sys.exit(1)
     try:
         config = Config(sys.argv[1:])
     # pylint: disable=broad-except
     except (BaseException) as be:
         print("Configuration invalid? %s" % (str(be)))
-        config.usage()
+        #config.usage()
```

Comment:
Nitpick: Either remove the line completely or use proper `# foo` in case there is value in keeping the content.